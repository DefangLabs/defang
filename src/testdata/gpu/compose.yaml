services:
  mistral:
    deploy:
      replicas: 1
      resources:
        reservations:
          devices:
            - capabilities: ["gpu"]
