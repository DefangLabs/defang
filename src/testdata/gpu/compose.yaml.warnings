 ! can only have more than one replica for PRO tier and PRODUCTION deployment mode. Will set replicas to 1 for service(s) [mistral]
 ! service "mistral": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors