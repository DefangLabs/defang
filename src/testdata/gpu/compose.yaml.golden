name: gpu
services:
  mistral:
    deploy:
      replicas: 1
      resources:
        reservations:
          devices:
            - capabilities:
                - gpu
              count: -1
    networks:
      default: null
networks:
  default:
    name: gpu_default
